inginious/common/custom_yaml.py

```diff
@@ -74,10 +76,18 @@ def _long_str_representer(dumper, data):
     def _default_representer(dumper, data):
         return _long_str_representer(dumper, str(data))
 
+    def _timestamp_representer(dumper, data):
+        formatted_date = data.strftime("%Y-%m-%dT%H:%M:%S.%fZ") if data != "" else None
+        # Ensure the year, month and day are always correctly formatted
+        formatted_date = formatted_date.replace(data.strftime("%Y"), "{:04d}".format(data.year), 1)
+        return dumper.represent_scalar('tag:yaml.org,2002:timestamp', formatted_date)
```

Comment:
A None value for formatted_date generates an error on the web page : "NoneType isnot subscriptable".
Removing this check because _timestamp_representer() is only applied to datetime objects.